### PR TITLE
MDCB 2.3 Release Notes

### PR DESCRIPTION
**Pull request by carolwkc**
## Types of changes

- [ ] Fixing typo (please merge to production)
- [ ] Documenting a new feature (please merge to production)
- [ ] Documentation for future release (please do not merge to production)
- [ ] Something else (please add if needs merging to production or not)

## Checklist

- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I have added a preview link.

### CODE REVIEW

**Review by tbuchaillot — APPROVED** [review]
lgtm!

**Comment by carolwkc** [comment]
@dcs3spp This is ready for publish. Thanks Simon!

File: tyk-docs/content/release-notes/mdcb-2.3.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+menu:
+  main:
+    parent: "Release Notes"
+weight: 254
+---
+
+## 2.3.0
+Release date: 2023-06-28
+
+MDCB 2.3.0 is an update for Tyk v5.1 API Definitions. We have also updated Go version to 1.19
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/mdcb-2.3.md]:
```suggestion
MDCB 2.3.0 is an update for compatibility for synchronisation with Tyk v5.1 API Definitions.
```

File: tyk-docs/content/release-notes/mdcb-2.3.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+---
+
+## 2.3.0
+Release date: 2023-06-28
+
+MDCB 2.3.0 is an update for compatibility for synchronisation with Tyk v5.1 API Definitions.
+
+### Updated
+- Update MDCB to Go 1.19
+- Update API schema for Tyk v5.1
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/mdcb-2.3.md]:
```suggestion
- Update for compatibility with API definitions for Tyk v5.1
```

**Review by dcs3spp — APPROVED** [review]
lgtm

**Pull request merged**